code/zFulpstationCode/fulpops_overwrite.dm

```diff
@@ -0,0 +1,55 @@
+/datum/antagonist/nukeop
+	name = "Nuclear Operative"
```

Comment:
You only need to redefine the specific vars you want to change, so most of this is presumably redundant.

Reply:
Cleaned up the code.